lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -188,11 +188,34 @@ defmodule ExUnit.Runner do
       {:error, %{test_case | state: failed}}
   end
 
-  defp run_test(config, test, context) do
+  defp run_test(true, config, test, context) do
+    run_test(:stdio, config, test, context)
+  end
+
+  defp run_test(false, config, test, context) do
+    spawn_test(config, test, context)
+  end
+
+  defp run_test(device, config, test, context) do
+    try do
+      logged =
+        ExUnit.CaptureLog.capture_log(device, fn ->
+          send self(), {:capture, spawn_test(config, test, context)}
+        end)
+      receive do
+        {:capture, test} -> %{test | logs: logged}
+      end
+    catch
+      :exit, :noproc -> %{test | state: {:skip, :no_logger}}
```

Comment:
Right, there is a comment about it https://github.com/elixir-lang/elixir/pull/3390#issuecomment-112985274. :smile:
By error mark you mean `:failed`, right? I think `:invalid` might be better for uset as it has more connection to the test preparation failures in contrast with `:failed`.